test/common/README.md

```diff
@@ -945,13 +945,15 @@ The realpath of the testing temporary directory.
 
 Deletes and recreates the testing temporary directory.
 
-The first time `refresh()` runs,  it adds a listener to process `'exit'` that
+The first time `refresh()` runs, it adds a listener to process `'exit'` that
 cleans the temporary directory. Thus, every file under `tmpdir.path` needs to
 be closed before the test completes. A good way to do this is to add a
 listener to process `'beforeExit'`. If a file needs to be left open until
 Node.js completes, use a child process and call `refresh()` only in the
 parent.
 
+**Note: When needed, call `refresh()` only once in each test file.**
```

Comment:
While it's often true that calling `refresh()` multiple times should be avoided, it's not always true. `test/sequential/test-heap-prof.js` is a counter-example.

Let's be sparing with the bold text etc. And let's also briefly explain why it should only be called once.

```suggestion
It is usually only necessary to call `refresh()` once in a test file. 
Avoid calling it more than once in an asynchronous context as one call
might refresh the temporary directory of a different context, causing
the test to fail somewhat mysteriously.
```

Reply:
Thanks, this is much better!